src/PopEngine/index.ts

```diff
@@ -64,14 +64,34 @@ export class PopEngine {
     }
   }
 
+  private _clearTimeoutByGroupId(groupId: string): void {
+    this._maybeClearTimeout(this._timeouts.timeToHoverOnPop, groupId);
+    this._maybeClearTimeout(this._timeouts.hoverdelay, null);
+  }
+
+  public clearTimeout(targetElement: Element): void {
+    let groupId = targetElement.getAttribute('popgun-group') || targetElement.getAttribute('pop-id');
+    this._clearTimeoutByGroupId(groupId);
+  }
+
+  public createPopElement(targetElement: Element): Element {
+    let container = document.createElement('div');
+    let nose = document.createElement('div');
+    container.setAttribute('class', 'pop');
+    container.setAttribute('pop-id', targetElement.getAttribute('popgun-group'));
+    container.setAttribute('pop', '');
+    nose.setAttribute('class', 'nose-triangle');
+    container.appendChild(nose);
+    return container;
```

Comment:
Ahhh, interesting. I will remove that unnecessary dependency by grabbing the node as needed (i.e when _setPosition is called). LMK if no bueno.